packages/component-library/components/Layout/Layout.tsx

```diff
@@ -1,119 +0,0 @@
-import * as React from "react"
```

Comment:
Same again here. I'm seeing 3 usages of the Layout component in our repos, one in each of these:
* murmur
* survey-configuration
* big-frontend-repo

Reply:
https://github.com/cultureamp/kaizen-design-system/pull/2739